src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -639,15 +676,17 @@ internal async Task Document_Change(BackgroundAnalysisScope analysisScope, bool
 
             var worker = await ExecuteOperation(workspace, w => w.ChangeDocument(document.Id, SourceText.From("//")));
 
+            var expectedDocumentEvents = 1;
```

Comment:
Why all these changes to tests? Do we now expect the same events in all the analysis modes?

Reply:
Yes. Incremental analyzers are now responsible for choosing whether or not to act (and the scope of actions) for each analysis mode.